data.sql

```diff
@@ -33,4 +33,60 @@ SET owner_id = CASE
 	WHEN name = 'Devimon' OR name = 'Plantmon' THEN 3
 	WHEN name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom' THEN 4
 	WHEN name = 'Angemon' OR name = 'Boarmon' THEN 5
-END;
\ No newline at end of file
+END;
+
+/* day 4 adding new values to the tables */
+
+INSERT INTO vets(
+        name,
+        age,
+        date_of_graduation
+      )
+        VALUES
+          ('Vet William Tatcher', 45, '2000/04/23'),
+          ('Vet Maisy Smith', 26, '2019/01/17'),
+          ('Vet Stephanie Mendez', 64, '1981/05/04'),
+          ('Vet Jack Harkness', 38, '2008/06/08')
+    ;
+
+  INSERT INTO specializations (species_id, vet_id)
+    VALUES
+      (1,1),
+      (1,3),
+      (2,3), 
+      (2,4)
+  ;
+
+  INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES
+      (6, 1, '2020-05-24'),
+      (6, 3, '2020-07-22'),
+      (3, 4, '2021-02-02'),
+      (9, 2, '2020-01-05'),
+      (9, 2, '2020-03-08'),
+      (9, 2, '2020-05-14'),
+      (4, 3, '2021-05-04'),
+      (7, 4, '2021-02-24'),
+      (1, 2, '2019-12-21'),
+      (1, 1, '2020-08-10'),
+      (1, 2, '2021-04-07'),
+      (8, 3, '2019-09-29'),
+      (2, 4, '2020-10-03'),
+      (2, 4, '2020-11-04'),
+      (5, 2, '2019-01-24'),
+      (5, 2, '2019-05-15'),
+      (5, 2, '2020-02-27'),
+      (5, 2, '2020-08-03'),
+      (10, 3, '2020-05-24'),
+      (10, 1, '2021-01-11')
+	;
+
+/* week 2 - day 1 */
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
+INSERT into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
+
+CREATE INDEX animal_id_index ON visits (animal_id);
+CREATE INDEX vet_id_index ON visits (vet_id);
+CREATE INDEX email_index ON owners (email);
```

Comment:
👍 Good progress on adding the indexes! However, there's a small adjustment to make to align with the curriculum guidelines.

- [ ] [Optional] Currently, your index creation operation is inside the `data.sql` file. According to the [curriculum-databases-projects-template](https://github.com/microverseinc/curriculum-template-databases#:~:text=This%20repository%20includes,Use%20them%20responsibly!), `CREATE` operations like index creation should be inside the `schema.sql` file. Kindly move these operations to the appropriate `schema.sql` file.

By organizing your SQL scripts this way, you'll ensure that your project structure aligns with the curriculum guidelines. Keep up the great work! 👍📜